tests/unit/test-util/test.adb.js

```diff
@@ -622,6 +632,35 @@ describe('utils/adb', () => {
        });
   });
 
+
+  describe('checkOrCleanArtifacts', () => {
+    it('cleans the previously artifacts directory when condition true' +
+      ' but not removed earlier', async () => {
+      const adb = getFakeADBKit({
+        adbClient: {
+          readdir: sinon.spy(() => Promise.resolve([file])),
+          shell: sinon.spy(() => Promise.resolve('')),
+        },
+        adbkitUtil: {
+          readAll: sinon.spy(() => Promise.resolve(Buffer.from('1\n'))),
+        },
+      });
+      const adbUtils = new ADBUtils({adb});
+      const temp = '/sdcard/';
+      const artifactsDir = temp + artifactDir;
```

Comment:
calling this `artifactsDir` doesn't explain what is different from `artifactDir` and this one, `artifactDirFullPath` would have been a better name (as it immediately suggests that the difference is that this one is a full absolute path).